addons/mail/static/src/discuss/core/public_web/discuss_sidebar_categories.js

```diff
@@ -143,13 +143,27 @@ export class DiscussSidebarChannel extends Component {
         return this.props.thread;
     }
 
+    get subChannels() {
+        return this.env.filteredThreads?.(this.thread.sub_channel_ids);
+    }
+
+    get isAnyChannelOpen() {
+        if (this.thread.eq(this.store.discuss.thread)) {
+            return true;
+        }
+        if (this.subChannels) {
+            return this.subChannels.some((subChannel) => subChannel.eq(this.store.discuss.thread));
+        }
+        return false;
+    }
+
     askConfirmation(body) {
         return new Promise((resolve) => {
             this.dialogService.add(ConfirmationDialog, {
                 body: body,
                 confirmLabel: _t("Leave Conversation"),
                 confirm: resolve,
-                cancel: () => {},
+                cancel: () => { },
```

Comment:
Useless diff

Reply:
for a fix PR, we should not introduce unrelated formatting change, that could introduced a lot of trouble later in the chain.